support-workers/src/main/scala/com/gu/support/workers/lambdas/FailureHandler.scala

```diff
@@ -60,8 +60,11 @@ class FailureHandler(emailService: EmailService) extends Handler[FailureHandlerS
     logger.info(s"Attempting to handle error $error")
     val pattern =
       "No such token: (.*); a similar object exists in test mode, but a live mode key was used to make this request.".r
-    val cardDeclinedMessage =
-      "Transaction declined.402 - [card_error/card_declined/do_not_honor] Your card was declined."
+    val cardDeclinedMessages = List(
+      "Transaction declined.402 - [card_error/card_declined/do_not_honor] Your card was declined.",
+      "Transaction declined.402 - [card_error/card_declined/insufficient_funds] Your card has insufficient funds.",
+      "Transaction declined.402 - [card_error/card_declined/try_again_later] Your card was declined.",
+    )
```

Comment:
Potentially we could just look for messages that start with 'Transaction declined.402' but I thought I'd stick with the current approach for a bit longer.